great_expectations/datasource/batch_kwargs_generator/batch_kwargs_generator.py

```diff
@@ -9,7 +9,7 @@
 
 
 class BatchKwargsGenerator(object):
-    """
+    r"""
```

Comment:
What does this do?

Reply:
It makes all your problems go away.

Try it.

Just once can't hurt.

(But the serious answer is that it makes python not treat `\` as an escape character, which is nice when you want to write a regex, because then you don't escape your escape. That doesn't *really* matter in this case since we don't operate on the docstring in general, but we *do* sometimes parse our own docstrings now for feature annotations, and it gets rid of warnings in the case of illegal would-be escaping.